ui/src/app/cohort-review/detail-page/detail-page.html

```diff
@@ -1,33 +1,41 @@
-<div class="detail-page" [class.sidebar-open]="sidebarOpen">
+<ng-container *ngIf="participant && annotations && annotationDefinitions">
```

Comment:
I observed if I click on participant from participants page, it is routing to the clicked participant and no spinner comes up, and it waits to get all data and then charts spinners start spinning. there is a time gap between getting chart data and all other data(like participants id and all). So I think we need to add a spinner here before ng-container , where we need to check ' <div class="spinner spinner-inverse" *ngIf="!participant && !annotations && !annotationDefinitions">Loading...</div>' -- something like this. but may be there will be more better way  to handle it. it was an observation.

Reply:
Good call, I added a spinner.

Reply:
nit : I think we need some css so that spinner will be in the center of the page. you can add this below css-
.review-spinner {
  position: absolute;
  top: 45%;
  left: 50%;
}
and add this class to the spinner div.